core/sys/cc.h

```diff
@@ -129,12 +129,16 @@
 
 #define CC_CONCAT2(s1, s2) s1##s2
 /**
- * A C preprocessing macro for concatenating to
- * strings.
+ * A C preprocessing macro for concatenating two strings.
  *
  * We need use two macros (CC_CONCAT and CC_CONCAT2) in order to allow
  * concatenation of two \#defined macros.
  */
 #define CC_CONCAT(s1, s2) CC_CONCAT2(s1, s2)
 
+/**
+ * A C preprocessing macro for concatenating three strings.
```

Comment:
Actually, it's not about strings. `##` is the C preprocessor token-pasting operator. So this macro and those above concatenate preprocessing tokens.